Description: Fix enable-ec2-spot-hibernation
 Debian and Ubuntu don't use chkconfig but update-rc.d and systemctl
 on newer releases.
Author: Balint Reczey <rbalint@ubuntu.com>
Forwarded: no
Last-Update: 2017-11-09

--- a/agent/enable-ec2-spot-hibernation
+++ b/agent/enable-ec2-spot-hibernation
@@ -3,7 +3,7 @@
 # for the EC2 Spot Instances.
 
 # Make sure the agent is added to the auto-start
-/sbin/chkconfig hibagent on
+update-rc.d hibagent defaults
 
 # And start it
 exec service hibagent start
